packages/framework/fluid-static/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # @fluidframework/fluid-static
 
+## 2.12.0
```

Comment:
Similar argument here, I don't think this change impacts fluid-static consumers so I think this could be "Dependency updates only".